src/websocket_service/handlers/notify_subscribe.rs

```diff
@@ -48,27 +58,63 @@ pub async fn handle(
         hex::encode(client_pubkey),
         project_data.signing_keypair.private_key,
     )?;
-    info!("[{request_id}] Response_sym_key: {}", &response_sym_key);
+    info!("[{request_id}] Response_sym_key: {}", &response_sym_key); // TODO don't log this
```

Comment:
Confused by the TODO, should be removed before deployment?

Reply:
It was already there, so I'm leaving it for now. Was planning to scan for more places where we log keys. But I could do this real quick in this PR.

Reply:
082ad28